roles/cis/tasks/main.yml

```diff
@@ -1,4 +1,5 @@
 ---
+- import_tasks: 1/task.yml
```

Comment:
There is a typo here (correct to 1/tasks.yml) due to which below error is coming:
1.30s$ docker run --privileged --rm  -v$(pwd):/src ansible-security-hardening:latest /src/examples/remediate_and_verify.sh
/src /
ERROR! Unable to retrieve file contents
Could not find or access '/src/1/task.yml' on the Ansible Controller.

Overall looks good to me.

However unless verification of these cases are added, the travis ci will mostly fail.